sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -176,6 +171,11 @@ export interface MessageHandlers<ReceivedMessageT> {
 
 export { MessagingError }
 
+// @public
+export interface OpenOptions {
```

Comment:
OpenOptions -> SenderOpenOptions ?

Otherwise, it feels too generic.

Also, perhaps for another time, I believe it is an overkill when we create interfaces to house just 1 property. 
Thoughts on inlining in such cases and adding an interface in the future when we have more properties to add?

cc @chradek, @bterlson

Reply:
I think I tend to prefer creating interfaces for options rather than in-line, unless it's a situation where you expect the user to pass in a class, and the interface just describes that class (e.g. inline an interface describing a method needed from ms-rest-nodeauth credentials).

The main reason being this gives someone an easy way to type options if they don't want to provide them inline with where they are being used.
But ultimately...what do the docs look like if we inlined the interface instead?

Reply:
Rename is fine (I believe I'm following the guideline here but it probably didn't anticipate having such simple method names :)). 

Let's leave the inlining discussion, as you suggested, for later. There is definitely something to be said for consistency.

Reply:
Renamed.